packages/plans-grid-next/src/components/shared/header-price/index.tsx

```diff
@@ -73,7 +73,7 @@ const HeaderPrice = ( { planSlug, visibleGridPlans }: HeaderPriceProps ) => {
 					/>
 					<PlanPrice
 						currencyCode={ currencyCode }
-						rawPrice={ introOffer.rawPrice.monthly }
+						rawPrice={ discountedPrice.monthly || introOffer.rawPrice.monthly }
```

Comment:
Sometimes, `discountedPrice.monthly` can be 0 if a 100% coupon is applied, and the discount won't be shown. Can we change the check to something like `typeof discountedPrice?. monthly === 'number'?`. 

(Related to https://github.com/Automattic/wp-calypso/pull/94579)

Reply:
Thanks for catching that, and the suggestion!
I applied it to  both cases and now 0 should be shown for these 100% coupons.
<img width="430" alt="Screenshot 2024-10-26 at 09 57 54" src="https://github.com/user-attachments/assets/42c71202-4bc5-402a-a1c8-a4fc5e369118">